liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/simple_page.html

```diff
@@ -2,6 +2,12 @@
 {% load wagtailcore_tags %}
 
 {% block content %}
-    <h1>{{ page.title }}</h1>
-    <p>{{ page.body|richtext }}</p>
+    <div class="l-wrapper">
+        <div class="l-center-6">
+            <div>
+                <h1>{{ page.title }}</h1>
+            </div>
+            <p>{{ page.body|richtext }}</p>
```

Comment:
I think `page.pody` should not be wrapped in a `<p>` because it contains block level html itself.

Reply:
I used the same code as in home_page template: https://github.com/liqd/a4-product/blob/master/liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html#L15

Reply:
Ok, we should change it in both places. I will do that.